releases/release-3.0.20.md

```
@@ -13,48 +13,48 @@ TiDB バージョン: 3.0.20
 
 -   TiDB
 
-    -   `enable-streaming`構成項目廃止する [＃21054](https://github.com/pingcap/tidb/pull/21054)
+    -   `enable-streaming`構成項目廃止する [#21054](https://github.com/pingcap/tidb/pull/21054)
```

markdownlint-cli2 (0.23.2)

[warning] Spaces after list markers
Expected: 1; Actual: 3

(MD030, list-marker-space)